ui/app/AppLayouts/Profile/views/WalletView.qml

```diff
@@ -79,6 +95,24 @@ SettingsContentBase {
         manageTokensView.resetChanges()
     }
 
+    readonly property var priv: QtObject {
+        id: priv
+        readonly property bool isManageTokensSubsection: Global.settingsSubSubsection === Constants.walletSettingsSubsection.manageAssets ||
+                                                         Global.settingsSubSubsection === Constants.walletSettingsSubsection.manageCollectibles ||
+                                                         Global.settingsSubSubsection === Constants.walletSettingsSubsection.manageHidden ||
+                                                         Global.settingsSubSubsection === Constants.walletSettingsSubsection.manageAdvanced
+
+        readonly property var assetSettings: Settings {
+            category: "AssetsViewSortSettings"
+            //property int currentSortValue
```

Comment:
Leftover?

Reply:
More of a comment for the reader; it's free to get removed when we refactor the settings/controller to some central store